### ISSUE TITLE
How is working the UNLOCKER & PAYOUT within private Pool ?

### ISSUE THREAD

**Issue by millenniumcm** [sammy007/open-ethereum-pool#385]
#### System information

OS & Version:  Ubuntu 16.04 LTS

#### Expected behaviour

Payout working properly to my workers

#### Actual behaviour

I have my own private ETH pool,  when i setup  the UNLOCKER & PAYOUT the same poolwallet,  the rewards remain in the wallet and workers are not being paid  automatically (i have to setup PAYOUT as "0x0" or empty in order to got workers paid automatically ?  i have 2 workers with 24GH/s in total, i am getting 2,6 - 3 eth each mined bloqued  and threshold is setup as 0,5 eth)

P.D. i have setup in PAYOUT different wallet from UNLOCKER and no worker got paid automatically

#### Steps to reproduce the behaviour

Just setup UNLOCKER & PAYOUT the same poolwallet

#### Backtrace

````
[backtrace]
````

**Comment by iRespectrum** [comment]
setup PAYOUT as  "0xWalletPool"

You need insert unlock wallet with 0x

**Comment by sammy007** [comment]
No need for that in private pools. Mine to your address and disable those modules.

**Comment by millenniumcm** [comment]
Hi @iRespectrum  , i have setup  the same POOL address  in UNLOCKER  & PAYOUT (with 0x..)   and no reward is sended to workers (just reamin in the pool,  and i want pool send it to my workers)

@sammy007  , So, i have to launch GETH command with --unlock  pointing to my worker address directly ? all my blocks are discovered by  the PoolWallet (current setup in UNLOCKER)

**Comment by mikeyb** [comment]
Yeah if you don't unlock the mining address in geth, it wont send payouts because it cant

**Comment by millenniumcm** [comment]
Ok  so,  i have  to --unlock  my worker address,  and  setup "false"  the Unlocker & Payout module..... but just one question..... in case if i have  2  worker address,  what address i must setup in Payout ? (or just leave empty  or with "0x0" ) ?

P.D. my main question is  if i setup Unlocker / Payout the same (pooladdress)  its remain in the pool, and don't send out to workers... i would like understand how would be the correct setup in order to automatically pool send payouts to workers addresses. (for public or private pools)

----------------------

Unlocker = 0xPoolWallet
Payout = 0xPoolWallet
Workers = No Payouts sended to Workers

----------------------

Unlocker = 0xPoolWallet
Payout = 0xExternalWallet
Workers = No Payouts sended to ExternalWallet

In both cases the rewards remains in the 0xPoolWallet (Pool Fee is 0,10%)  and of course both modules are setup as "true".

**Comment by millenniumcm** [comment]
Hello @mikeyb  could you install Pool by a Fee ? pls contact with us,  thanks
